src/main/java/module/workflow/domain/WorkflowProcess.java

```diff
@@ -1014,6 +1015,29 @@ public boolean hasUnreadCommentsForUser(final User user) {
         return getFilesSet();
     }
 
+    public void delete() {
+        for (WorkflowLog log : this.getExecutionLogsSet()) {
+            log.delete();
+        }
+        for (WorkflowProcessComment comment : this.getCommentsSet()) {
+            comment.delete();
+        }
+        for (WorkflowQueue queue : this.getCurrentQueuesSet()) {
+            queue.delete();
+        }
+        for (WorkflowQueue queue : this.getQueueHistorySet()) {
+            queue.delete();
```

Comment:
Queues should not be deleted. Thay may be used by other processes.
They should only be disconnected from the process being deleted.